lib/mix/lib/mix.ex

```diff
@@ -172,25 +172,31 @@ defmodule Mix do
       echo "world!"
 
       # mix.exs
-      defp create_aliases do
+      defp aliases do
         [
-          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
-          "taskalias2": ["run priv/hello1.exs", "run priv/hello2.exs"]
+          task_alias: ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
+          task_alias2: ["run priv/hello1.exs", "run priv/hello2.exs"]
         ]
       end
 
-  In the example above we have created 2 aliases, the first example
-  `taskalias` will run task `hex.info`, then (`run`)[`Mix.Tasks.Run`]
-  to run an Elixir script, then (`cmd`)[`Mix.Tasks.Cmd`] to run a
-  command line bash script. This shows how powerful aliases mixed
-  with mix tasks can be.
+  In the example above, we have created two aliases. In the first example,
+  `task_alias` will run the task `hex.info`, then [`run`](`Mix.Tasks.Run`)
+  to run an Elixir script, and then [`cmd`](`Mix.Tasks.Cmd`) to execute a
+  command line shell script. This shows how powerful aliases mixed
+  with Mix tasks can be.
 
-  `taskalias2` shows a limitation of tasks where only one of the given
-  tasks will run, the execution of `run priv/hello2.exs` will not run.
+  `task_alias2` shows a limitation where only the first of the given
+  tasks will run; the execution of `run priv/hello2.exs` will not run.
+  This comes from a design choice to only allow a task to be executed just once.
   The `run` command, however, can accept multiple files, so in case
   of running multiple files, it can be rewritten to:
 
-      "taskalias2": ["run -r priv/hello1.exs -r priv/hello2.exs"]
+      task_alias2: ["run -r priv/hello1.exs -r priv/hello2.exs"]
+
+  Conversely, you can always call [`Mix.Task.reenable "name_of_your_task"`](Mix.Task.reenable/1)
```

Comment:
Use parens here (like the formatter would).

Reply:
fixed